cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -80,9 +80,9 @@ class CallTest extends DocumentingTest {
 
     section("View the signature for a procedure", "call-view-the-signature-for-a-procedure") {
       p("""To `CALL` a procedure, its input parameters need to be known, and to use `YIELD`, its output parameters need to be known.
-          #The built-in procedure `dbms.procedures` returns the name, signature and description for all procedures.
+          #Use SHOW PROCEDURES which returns the name, signature and description for all procedures.
```

Comment:
do we need to mention anything about it not being the only fields returned? and that the signature is one of the verbose columns and not in the default output?

Reply:
Hmm, maybe. But it might be enough with the example below where they can see how to use it. What do you think?

Reply:
might be good to at least mention that it isn't among the default columns? so people don't do `SHOW PROCEDURES` and get confused that the signature isn't there?